contracts/mocks/MockFunctions.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.19;
+
+contract MockFunctions {
```

Comment:
Please add a contract level comment indicating that this file is part of the test code for GuardedMulticaller.

Reply:
done